transform-filename.js

```diff
@@ -10,6 +10,9 @@ const types = {
   'linux-armv6l': 'linux-armv6l',
   'linux-x64': 'linux-x64',
   'linux-x86': 'linux-x86',
+  'linux-x64-musl': 'linux-x64-musl',
```

Comment:
Does it make sense to call it out here?
```suggestion
  'linux-x64-musl': 'linux-x64-musl', // unofficial builds
```